rules/data/embedded/embedded-svg.yara

```diff
@@ -0,0 +1,46 @@
+rule svg: harmless {
+  meta:
+    description = "Contains SVG (Scalable Vector Graphics) content"
+
+  strings:
+    $svg_open1  = "<svg>"
+    $svg_open2  = "<SVG>"
+    $svg_close1 = "</svg>"
+    $svg_close2 = "</SVG>"
+
+  condition:
+    all of them
+}
+
+rule foreign_object: high {
+  meta:
+    description = "Contains SVG (Scalable Vector Graphics) content using foreignObjects"
+
+  strings:
+    $foreign_obj_open  = "<foreignObject>" nocase
+    $foreign_obj_close = "</foreignObject>" nocase
+
+  condition:
+    svg and all of them
+}
+
+rule foreign_object_script: critical {
```

Comment:
This feels more like a `high` to me; I think we'll find at least one false-positive for it in the future.

Reply:
Updated in [`19224b3` (#704)](https://github.com/chainguard-dev/malcontent/pull/704/commits/19224b38d27b23d94c32d35dda2d5b3bebfe0121).